demo/lib/TransformationControls/addLeadIntroductionControl.js

```diff
@@ -2,18 +2,24 @@
 
 /* ArticleRefSourceType is global */
 
-export default [
-  'first_paragraph_checkbox',
-  'Move 1st Paragraph Up *',
-  'checkbox',
-  null,
-  (iframeWindow, iframeDocument) => {
-    event.target.disabled = true
-    // eslint-disable-next-line no-undef
-    if (iframeDocument.articleRef.sourceType === ArticleRefSourceType.mobileView) {
+const moveLeadIntroductionUp = (iframeWindow, iframeDocument) => {
+  event.target.disabled = true
+  // eslint-disable-next-line no-undef
+  if (iframeDocument.articleRef.sourceType === ArticleRefSourceType.mobileView) {
+    if (iframeWindow.pagelib.c1) {
+      // TODO: moveLeadIntroductionUp is not exposed in the abstraction layer
```

Comment:
We're not going to add this AFAIK. The service side already does this, so there is no reason to expose it.